activestorage/lib/active_storage/attached/many.rb

```diff
@@ -62,5 +63,10 @@ def detach
     # :method: purge_later
     #
     # Purges each associated attachment through the queuing system.
```

Comment:
@zzak related to https://github.com/rails/rails/pull/42383#discussion_r648047086 I've noted something interesting: the RDOC for `purge_later` is no longer generated correctly after adding the `private` signature below.
I think this is a bug in RDOC, if that's the case I'll open an issue there.
As temporary workaround we can put a placeholder comment below (which leads into generating the RDOC correctly) 

WDYT? 🤔

Reply:
@intrip I think you need to add the `# :doc:` before the doc due to the change in method visibility:
https://guides.rubyonrails.org/api_documentation_guidelines.html#method-visibility

Reply:
@zzak I'm sorry but I don't get you 😞 : the `purge_later` is a public method, the visibility change occurs for the `private` method below it `purge_many` so I don't understand how `:doc:` would work. In any case I've followed your suggestion and put `:doc:` before the `purge_later` doc but It didn't work, I'm probably missing something...

The only way I've managed to make it working so fare are to either put a `#`  comment before the `purge_later` doc or move the doc upper in the source code.

Reply:
@intrip So two things here:

* If you want to document a private method I thought this was the purpose for `:doc:`
* As I commented below, I'm not sure we should remove this method if it is already public API

Even if no Rails code is depending on it, we don't know if someone _out in the wild_ is using it.

Reply:
@zzak sorry for the back and forth but I probably didn't explain myself well enough and I think we are are not on the same track:
- this method is not actually removed, is just exposed with `delegate` API (line 6 of this file)
- this method is public, not private
- the problem is that since we used delegate I removed the original method definition (`def purge_later`) hence in order to still show the docs I had to explicitly use `:method:`

Now: the doc for `:purge_later` for some reasons is not shown even if we use `:method:` signature (whilst the doc for :purge is shown correctly), hence I wrote https://github.com/rails/rails/pull/42383/files#r648092283

We can get rid of delegate api, keep the `def purge` and manually delegate to solve this, but I'm not sure this is the right approach...

I hope it's all clear now.

Reply:
@intrip That makes sense now, sorry for the extensive discussion for such a simple issue! 🤦 

One idea I have is to split up the `deletgate` calls to separate lines and try putting the RDoc above each definition. Something like:

```ruby
# :method: purge
# blah blah
delegate :purge, to: :purge_many

# :method: purge_later
# blah blah
delegate :purge_later, to: purge_many
```

Reply:
good idea! I've done that.